### PR TITLE
Zijian/change email to description

### LINKED ISSUE #119: Change Email field to Description

### PR DESCRIPTION
**Pull request by BobbyZhouZijian**
fixes #119 

Description field should not be empty, should not be blank or start with a blank. should contain only alphenumerical values, commas, periods, question marks or exclamation marks.

### CODE REVIEW

File: docs/tutorials/RemovingFields.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ private Label address;
 ...
 <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
 <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
-<Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+<Label fx:id="description" styleClass="cell_small_label" text="\$description" />
```
Comment (MarcTzh) [inline docs/tutorials/RemovingFields.md]: Don't edit the tutorial

File: docs/tutorials/RemovingFields.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ In `src/test/data/`, data meant for testing purposes are stored. While keeping t
   "tasks": [ {
     "name": "Person with invalid name field: Ha!ns Mu@ster",
     "phone": "9482424",
-    "email": "hans@example.com",
+    "description": "task with invalid description: 123!$@#%@%@#",
```
Comment (MarcTzh) [inline docs/tutorials/RemovingFields.md]: ^

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 3]
```diff
@@ -81,18 +81,18 @@ public static Address parseAddress(String address) throws ParseException {
...
     /**
-     * Parses a {@code String email} into an {@code Email}.
+     * Parses a {@code String description} into an {@code Description}.
      * Leading and trailing whitespaces will be trimmed.
      *
-     * @throws ParseException if the given {@code email} is invalid.
+     * @throws ParseException if the given {@code description} is invalid.
      */
-    public static Email parseEmail(String email) throws ParseException {
+    public static Description parseDescription(String email) throws ParseException {
```
Comment (MarcTzh) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: Parameter name not changed

File: src/test/data/JsonSerializableAddressBookTest/typicalTaskAddressBook.json [hunk 4]
```diff
@@ -3,43 +3,43 @@
   "tasks" : [ {
     "title" : "Alice Pauline",
     "phone" : "94351253",
-    "email" : "alice@example.com",
+    "description" : "alice,example.com",
```
Comment (MarcTzh) [inline src/test/data/JsonSerializableAddressBookTest/typicalTaskAddressBook.json:6]: Good

**Review by MarcTzh — COMMENTED** [review]
Change one parameter, other than that looks good

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 3 above)
Comment (BobbyZhouZijian) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java]: nice catch

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 5]
```diff
@@ -41,16 +41,16 @@
...
+    public static final String DESCRIPTION_DESC_BOB = " " + PREFIX_DESCRIPTION + VALID_DESCRIPTION_BOB;
     public static final String ADDRESS_DESC_AMY = " " + PREFIX_ADDRESS + VALID_ADDRESS_AMY;
     public static final String ADDRESS_DESC_BOB = " " + PREFIX_ADDRESS + VALID_ADDRESS_BOB;
     public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
 
     public static final String INVALID_TITLE_DESC = " " + PREFIX_TITLE + "Homework&"; // '&' not allowed in titles
     public static final String INVALID_PHONE_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
-    public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
+    public static final String INVALID_DESCRIPTION_DESC = " " + PREFIX_DESCRIPTION + "bob@yahoo"; // using '@' symbol
```
Comment (MarcTzh) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java]: uses '@' symbol

**Review by MarcTzh — APPROVED** [review]
One minor comment suggestion, other than that LGTM

**Pull request merged**